### ISSUE TITLE
Multi-lang sitemap

### ISSUE THREAD

**Issue by CrystalMethod** [getgrav/grav-plugin-sitemap#7]
Having a multi-lang site I would expect to see entries for each supported language (``en``, ``de``) but only see ``en``.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
        <url>
                <loc>.../en</loc>
                <lastmod>2015-07-29</lastmod>
                <changefreq>monthly</changefreq>
                <priority>1.0</priority>
        </url>
        <url>
                <loc>.../en</loc>
                <lastmod>2015-07-29</lastmod>
                <changefreq>monthly</changefreq>
                <priority>1.0</priority>
        </url>
        <url>
                <loc>.../en/...</loc>
                <lastmod>2015-07-29</lastmod>
        </url>
        <url>
```

**Comment by rhukster** [comment]
This plugin needs to be reworked to support multilang

**Comment by cord** [comment]
would be great to have multilang support :)

**Comment by teikko** [comment]
This would be great. :)

Also support for linking other sitemaps e.g. sub domain hosted outside GRAV would be very handy.

https://support.google.com/webmasters/answer/75712?hl=en&ref_topic=4581190

**Comment by cord** [comment]
actually it works already, see:

http://www.mailbeez.de/sitemap.xml
http://www.mailbeez.com/sitemap.xml

**Comment by trial-n-error** [comment]
Hi,

for me it seems multilang support is not yet fully supported. Might be also I found a bug.

In my multilang configuration I support two languages: 'en' and 'de'. I configured for the default 'en' that "include_default_lang" is "false".

When rendering the 2nd language sitemap:

<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/xsl" href="/de/user/plugins/sitemap/sitemap.xsl"?>
...

For me it seems that the path to the xsl file should not be prefixed with the 2nd language abbreviation. Other than that it seems ok for me.

Best regards,
 Mario
